Jenkinsfile

```diff
@@ -1,4 +1,7 @@
-String[] pathArray = ["src/"]
+pathArray = [
```

Comment:
In that case initial name `pathArray` is wrong because it suggest that we put there all paths, didn't it?
More appropriate name would be `srcPaths`.
For log directory I'd recommend `logMainPath`.